Source/JavaScriptCore/runtime/ISO8601.cpp

```diff
@@ -1680,20 +1674,84 @@ std::optional<ExactTime> ExactTime::add(Duration duration) const
     return result;
 }
 
-Int128 ExactTime::round(Int128 quantity, unsigned increment, TemporalUnit unit, RoundingMode roundingMode)
+// https://tc39.es/proposal-temporal/#sec-temporal-roundtemporalinstant
+static Int128 roundTemporalInstant(Int128 ns, unsigned increment, TemporalUnit unit, RoundingMode roundingMode)
+{
+    auto unitLength = lengthInNanoseconds(unit);
+    auto incrementNs = increment * unitLength;
+    return roundNumberToIncrementAsIfPositive(ns, incrementNs, roundingMode);
+}
+
+// https://tc39.es/proposal-temporal/#sec-validatetemporalroundingincrement
+static void validateTemporalRoundingIncrement(JSGlobalObject* globalObject, unsigned increment,
+    Int128 dividend, bool inclusive)
```

Comment:
On other PRs I've gotten the feedback that something like `enum { Inclusive, NotInclusive }` is preferred instead of a boolean. Not sure if that's the case here.

Reply:
Changed in [1a7478f](https://github.com/WebKit/WebKit/pull/45603/commits/1a7478ffedbc22e97ccd46a002d9b3f7cc8072a9)